vector/src/main/res/layout/keybackup_setup_step2_fragment.xml

```diff
@@ -0,0 +1,151 @@
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="16dp"
+                    android:layout_marginLeft="16dp"
+                    android:baselineAligned="false"
+                    android:orientation="horizontal">
+
+                    <android.support.design.widget.TextInputLayout
+                        android:id="@+id/keybackup_passphrase_enter_til"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:layout_weight="1"
+                        app:errorEnabled="true">
+
+                        <android.support.design.widget.TextInputEditText
+                            android:id="@+id/keybackup_passphrase_enter_edittext"
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:hint="@string/keybackup_setup_step2_passphrase_enter_hint"
+                            android:maxLines="3"
+                            android:textColor="?android:textColorPrimary" />
+
+                    </android.support.design.widget.TextInputLayout>
+
+                    <RelativeLayout
+                        android:id="@+id/keybackup_view_password_clickable"
+                        android:layout_width="44dp"
+                        android:layout_height="match_parent"
+                        android:background="?attr/selectableItemBackground">
+
+                        <ImageView
+                            android:layout_width="22dp"
+                            android:layout_height="18dp"
+                            android:layout_centerHorizontal="true"
+                            android:layout_centerVertical="true"
+                            android:src="@drawable/ic_eye_black"
+                            android:tint="?attr/colorAccent" />
+                    </RelativeLayout>
+
+                </LinearLayout>
+
+                <im.vector.ui.PasswordStrengthBar
+                    android:id="@+id/keybackup_passphrase_security_progress"
+                    android:layout_width="match_parent"
+                    android:layout_height="4dp"
+                    android:layout_marginStart="16dp"
+                    android:layout_marginLeft="16dp"
+                    android:layout_marginEnd="44dp"
+                    android:layout_marginRight="44dp" />
+
+                <View
+                    android:layout_width="match_parent"
+                    android:layout_height="1dp"
+                    android:layout_margin="6dp"
+                    android:background="@color/list_divider_color_light" />
```

Comment:
You must use `?attr` and not directly color, to correctly support all the themes

Reply:
done